src/kit/Section.module.scss

```diff
@@ -0,0 +1,30 @@
+/* stylelint-disable no-descending-specificity */
+
+.base {
+  &:not(:last-child) {
+    padding-bottom: 16px;
+  }
+  .header {
+    border: 0 solid var(--theme-stage-border);
```

Comment:
q: why define the border here instead of in the .titleDivider selector at the bottom?